src/App.vue

```diff
@@ -127,6 +127,24 @@
       socket.on('server:state', function () {
         updateState()
       })
+      socket.on('server:title-round-over', function () {
+        updateState(() => {
+          console.log('Story Title Picked')
+          // If I won the last round, clear my suggestion box
+          assert(self.sharedState.story)
+          assert(self.sharedState.story.title)
+          var lastRound = self.sharedState.story.title
```

Comment:
Using `var` is not good here, it collides with another variable with the same name (scope of var = function scope).

1. I changed lastRound here to titleRound
2. I changed `var` to `let`